submissions/xT0PERx/js-dom/code.js

```diff
@@ -0,0 +1,65 @@
+var contentArr = [
```

Comment:
https://www.robinwieruch.de/javascript-naming-convention
Check out this variable naming guide

Reply:
Thank. Unfortunately, by clicking on the link, I found the inscription: "Page not found".

Reply:
fix link =)